pkg/operator/errors/errors.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2019 Cortex Labs, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package errors
+
+import (
+	"github.com/cortexlabs/cortex/pkg/lib/errors"
+	"github.com/cortexlabs/cortex/pkg/operator/telemetry"
+)
+
+func ReportAndExit(items ...interface{}) {
+	err := errors.ConsolidateErrItems(items...)
+	telemetry.ReportErrorBlocking(err)
+	errors.Exit(err)
+}
+
+func ReportAndRecover(strs ...string) error {
+	if errInterface := recover(); errInterface != nil {
+		err := errors.CastRecoverError(errInterface, strs...)
+		telemetry.ReportErrorBlocking(err)
```

Comment:
Can this be asynchronous?

Reply:
Okay will make it async and add Cron to the function title